enhancements/oc/mirroring-manifests.md

```diff
@@ -105,78 +98,104 @@ mirror
+This filesystem can then be applied to the local registry with the given `oc image mirror` command.
 
-With this enhancement, the output format would gain an additional directory `manifests` as a sibling to the current `v2` containing manifest files that can be applied to the local cluster with:
+With this enhancement, the output format would gain an additional directory `manifests` under the base directory, a sibling to the current `v2`, containing manifest files that can be applied to the local cluster with:
 
 ```console
 $ oc apply -Rf mirror/manifests
 ```
 
 The `manifests` directory may or may not contain additional subdirectories, so the recursive `-R` should be given for future-proofing.
 
-### Pushing to disk while pushing containers to the mirror registry
-
-Users mirroring a release before creating a cluster may want to [push both container images and manifests to disk](#pushing-to-disk).
-But they might also want to push container images directly to a mirror repository (as in [the *applying directly to the target cluster* case](#applying-directly-to-the-target-cluster) while still pushing manifests to disk (because there is not yet a cluster into which those manifests can be pushed).
-While you could address this use-case with [the *pushing to disk* flow](#pushing-to-disk) followed by an immediate `oc image mirror ...`, it is more convenient to have `oc adm release mirror ...` push to the registry directly.
+Users may also want to push container images directly to a mirror repository (as in the example command in [the *Summary*](#summary) while still pushing manifests to disk (because there is not yet a cluster into which those manifests can be pushed).
+While you could address this use-case with the command above followed by an immediate `oc image mirror ...`, it is more convenient to have `oc adm release mirror ...` push to the registry directly.
 Because setting `--to-dir` adjusts the image source, this enhancement extends the command with a new `--manifests-to-dir` that can be used to set a manifest output directory without setting a container image output directory.
 When `--manifests-to-dir` is set, it takes precedence.
-When `--manifests-to-dir` is unset but `--to-dir` is set, the manifest directory is `${TO_DIR}/manifests`, as described in [the previous section](#pushing-to-disk).
+When `--manifests-to-dir` is unset but `--to-dir` is set, the manifest directory is `${TO_DIR}/manifests`, as described above.
+
+### Applying manifests directly to the target cluster
+
+This enhancement adds a new `--apply-manifests` option that when specified will apply manifests directly to the connected cluster rather than outputting them to disk.
+When the `--apply-manifests` option is specified a user can also optionally specify `--overwrite` which would cause the apply to bahave as the 'oc apply --overwrite' does currently. If the manifest exists on the cluster it will be updated.
```

Comment:
nits: "bahave" -> "behave" and backticks instead of single-quotes for `oc apply --overwrite`.  Also, two sentences on one line.

Reply:
will fix